tests/foreman/cli/test_satellitesync.py

```diff
@@ -2139,30 +2112,171 @@ def test_positive_custom_cdn_with_credential(
             repo['content-counts'] == function_synced_rh_repo['content-counts']
         ), 'Content counts do not match'
 
-    @pytest.mark.stubbed
+    @pytest.mark.e2e
     @pytest.mark.tier3
-    @pytest.mark.upgrade
-    def test_positive_install_package_from_imported_repos(self):
-        """Install packages in client from imported repo of Downstream satellite.
+    @pytest.mark.rhel_ver_list([8])
+    @pytest.mark.parametrize(
+        'function_synced_rh_repo',
+        ['rhae2.9_el8'],
+        indirect=True,
+    )
+    def test_positive_export_import_consume_incremental_yum_repo(
+        self,
+        target_sat,
+        export_import_cleanup_function,
+        config_export_import_settings,
+        function_sca_manifest_org,
+        function_import_org_with_manifest,
+        function_synced_rh_repo,
+        rhel_contenthost,
+    ):
+        """Export and import RH yum repo incrementally and consume it on a content host.
 
-        :id: a81ffb55-398d-4ad0-bcae-5ed48f504ded
+        :id: f5515168-c3c9-4351-9f83-ba6265689db3
 
-        :steps:
+        :setup:
+            1. Enabled and synced RH yum repository (Ansible Engine for this case).
+            2. An unregistered RHEL8 host.
 
-            1. Export whole Red Hat YUM repo to a path accessible over HTTP.
-            2. Import the Red Hat repository by defining the CDN URL from the
-               exported HTTP URL.
-            3. In downstream satellite create CV, AK with this imported repo.
-            4. Register/Subscribe a client with a downstream satellite.
-            5. Attempt to install a package on a client from imported repo of
-               downstream.
+        :steps:
+            1. Create a CV with the RH yum repository.
+            2. Add exclude RPM filter to filter out one package, publish version 1 and export it.
+            3. On the importing side import version 1, check the package count.
+            4. Create an AK with the imported CV, register the content host and check
+               the package count available to install. Filtered package should be missing.
+            5. Update the fiter so no package is left behind, publish version 2 and export it.
+            6. Import version 2, check the package count.
+            7. Check the package count available to install on the content host.
+            8. Install the package.
 
         :expectedresults:
-            1. The package is installed on client from imported repo of downstream satellite.
+            1. More packages available for install after version 2 imported.
+            2. Packages can be installed successfully.
 
-        :CaseAutomation: NotAutomated
+        :BZ: 2173756
 
+        :customerscenario: true
         """
+        res = rhel_contenthost.execute('dnf config-manager --set-disabled ubi-8-appstream-rpms')
+        assert res.status == 0, 'disablement of ubi-8-appstream-rpms failed'
```

Comment:
Decided to replace the repo used in this case with one not-interfering with ubi-8.

Reply:
For the assertion, now it would have to be `assert pkg_cnt_2 == pkg_cnt_1 + 2` since there are **currently** two versions of the filtered package. I don't think we should fail the test here when the count of the versioned package change.

Reply:
@vsedmik, makes sense. Thanks for addressing this